src/main/java/com/mk/cti/controller/PhoneController.java

```diff
@@ -27,8 +30,7 @@
 
         PhoneResponse phoneResponse = phoneResponseService.findByPhoneNumber(phoneNumber);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Line is longer than 80 characters (found 90). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck)